### ISSUE TITLE
Deliverables for CUP copy editing

### ISSUE THREAD

**Issue by ebridge2** [neurodata/graph-stats-book#111]
Labels:
The axis labels in many figures (e.g., 5.4.1) are fuzzy or clipped. Has the PDF manuscript been downsampled? If not, these figures probably need to be regenerated.
- [x] 2.6.1: size of left image should be increased relative to right image so that the labels in the left image are more readable
- [x] 3.4.1: make operators smaller and boxes bigger; the figure will be clearer that way.
- [x] 6.6.1 C: fix overlapping labels
- [x] 6.2.1: please lay this out differently or relabel so that labels are larger and more readable
- [x] 9.1.5: please make the smaller labeling larger if you can
- [ ] 9.1.6, 9.1.7, 9.1.9: the identifiers A, B, C should be made smaller since the figures will overall be getting larger 

Color figures:
- [ ] Are these in CMYK? If not, please convert them to CMYK.
- [x] 2.4.2, 2.5.1 – The yellow is too faint for good printing, and the color dots in the key are very small
- [x] 6.2.3 - similar problem with small color dot key
- [x] Red/Green figures: The combination of red and green in figures can’t be distinguished by color blind people. Please recolor the following figures in some other combination of colors (eg red and blue): 1.2.5 (this should be redrawn in b/w) 1.3.1, 1.3.2, 1.3.4, 3.0.1, 4.0.1, 5.0.1, 5.6.1, 5.6.2, 6.1.3, 6.1.4
 

Fonts:

- [ ] What is the sans serif font used in the manuscript? I can check if we have a license for it. If we do not, we’ll switch it to Helvetica. (The typesetter will change the font of the main text to Times.)
 

Manuscript:

Please move the below sections to online only. They are more useful online than in print and this will help bring down the page count.
- [x] Example and application from Appendix B.3
- [x] Appendix C: put everything except the Silhouette online
- [x] Appendix B2 is blank. Please fix/remove.

We will remove lists of figures, tables, boxes, and contributors from the frontmatter. The author names will go on title page, and contributor names in the table of contents.
- [x] We will need the contributors to sign contributor agreements. Please provide their contact details and the specifics of their contribution so we can draft the agreements.
- [x] First, regarding the title: You may recall we discussed this extensively at the time the contract was signed and agreed on the title “Hands-On Network Machine Learning in Python: with Scikit Learn and graspologic”. Both our marketing and sales team and my faculty governing board felt very strongly that the main title should still start with “Hands-On”, because that best describes the presentation. It should be clear from the outset that this is a practical rather than a theoretical book; we want to attract the readers looking for the former without alienating readers looking for the latter. The title also needs to indicate the programming language if not all the specific packages. So we could consider dropping the subtitle, which might get unwieldy if we include all of NumPy, SciPy, scikit-learn, graspologic, and NetworkX (although there are definitely precedents for this, see https://www.amazon.com/Hands-Machine-Learning-Scikit-Learn-TensorFlow/dp/1098125975). But “Python” definitely needs to be in the title.
- [x] Section title boxes: this material is really pretty standard for section introductions and should just be part of the text proper. There is no reason to put this material in boxes. There are more than enough boxes in the text already. Having the section title in a box will be quite confusing to readers. The gray shading is unappealing (as you point out), plus it is hard to read black text on gray shading.
